packages/store/src/state-factory.ts

```diff
@@ -114,20 +119,45 @@ export class StateFactory {
     }
   }
 
+  connectActionHandlers(mappedStores: MappedStore[]): any {
+    if (this._connected) return;
+    this._actions
+      .pipe(
+        filter((ctx: ActionContext) => ctx.status === ActionStatus.Dispatched),
+        mergeMap(({ action }) => {
+          return this.invokeActions(this._actions, action).pipe(
+            map(() => {
+              return <ActionContext>{ action, status: ActionStatus.Completed };
+            }),
+            defaultIfEmpty(<ActionContext>{ action, status: ActionStatus.Canceled }),
+            catchError(err => {
+              return of(<ActionContext>{ action, status: ActionStatus.Errored });
+            })
+          );
+        })
+      )
+      .subscribe(ctx => {
```

Comment:
Make this a lambda.

Reply:
Done, although I prefer having them in a statement block because you can then put a breakpoint there if necessary.